plugins/utils/redirects.js

```diff
@@ -432,8 +424,6 @@ const redirects = `
   /docs/current/developer-docs/integrations/https-outcalls/https-outcalls-how-to-use /docs/current/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-how-to-use
   /docs/current/developer-docs/integrations/https-outcalls/https-outcalls-overview /docs/current/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-overview
   /docs/current/developer-docs/integrations/https-outcalls/https-outcalls-post /docs/current/developer-docs/smart-contracts/advanced-features/https-outcalls/https-outcalls-post
-  /docs/current/developer-docs/integrations/bitcoin/ckBTC-reference /docs/current/references/ckbtc-reference
```

Comment:
These ckBTC redirects are necessary since they redirect case-sensitive URLs to the correct lowercase URLs.

Reply:
I should have only deleted a duplicate. I'll double check

Reply:
yep, I only deleted a duplicate. `/docs/current/developer-docs/integrations/bitcoin/ckbtc-reference /docs/current/references/ckbtc-reference` is still there

Reply:
We still need the redirects for `/docs/current/developer-docs/integrations/bitcoin/ckBTC-reference` and `/docs/current/tutorials/developer-journey/level-4/4.3-ckBTC-and-bitcoin`  since this URL is case-sensitive. Slack thread about this here: https://dfinity.slack.com/archives/C03856W4LRG/p1708731304240029

Reply:
🤔 That's weird. I guess let's remove them for now so the build passes, and I can take a closer look at re-adding them in the future.